src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/VariableMultiValueConverter.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class VariableMultiValueConverter : MultiValueConverterExtension, IMultiValueConverter
+	{
+		public MultiBindingCondition ConditionType { get; set; }
+
+		public int Count { get; set; }
+
+		public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (values == null)
```

Comment:
```suggestion
			if (values == null || values.Length== 0)
```

Reply:
This one creates kind of a situation. Let's say you have a variable list of values, and you want `None` of the values to be true. However, it could also be that the list has no values. Then `None` would theoretically be true as there are no booleans in there that are true. I wouldn't personally add length-based escapes and allow for the list to also be empty and continue functioning as it does.

Reply:
I didn't see this use case. I think it's ok to keep the current implementation.